src/DBus/DBusBridge.hpp

```diff
@@ -28,6 +28,14 @@
 #include <gio/gio.h>
 #pragma clang diagnostic pop
 
+#define DBUS_BUS_NAME          "org.usbguard1"
+#define DBUS_ROOT_INTERFACE    "org.usbguard1"
+#define DBUS_ROOT_PATH         "/org/usbguard1"
+#define DBUS_POLICY_INTERFACE  "org.usbguard1.Policy"
```

Comment:
@tweksteen What still confuses me:
```
"org.usbguard1.Policy"
  Why here --^       ^--- and not here?
```

Reply:
Right, I agree. Re-read the blog post and with @muelli's explanation that make sense.